hack/update-codegen.sh

```diff
@@ -77,6 +77,7 @@ ${GOPATH}/bin/deepcopy-gen \
   -i knative.dev/serving/pkg/reconciler/gc/config \
   -i knative.dev/serving/pkg/reconciler/revision/config \
   -i knative.dev/serving/pkg/reconciler/route/config \
+  -i knative.dev/serving/pkg/reconciler/service/config \
```

Comment:
I'd lump this into the second planned change vs. adding a config store with nothing in it.